### DISCUSSION TITLE
There was a problem saving your key passkey

### DISCUSSION THREAD

**Discussion by scriptswhisper** [community/community#88701] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Hello,
I don't know if I'm in the right topic, but I'm facing the issue "there was a problem saving your key passkey" after my new mobile device.
The last one died so I've no way to recover passkey from it, I've deleted the old passkey from the github account settings but I'm unable to add my current mobile device as passkey.

Mobile device: oppo a98
Laptop os: win 11
Browser tested: chrome, firefox, edge

Is there any way to fix it?

![image](https://github.com/community/community/assets/135032973/d720a397-1742-4c29-8ed4-834222220b47)

![image](https://github.com/community/community/assets/135032973/0b99efd5-2ac6-4bd4-bbd3-1785bb82ba5f)

[error_github-com-har.zip](https://github.com/community/community/files/14053585/error_github-com-har.zip)

**Comment by B0K1NG** [comment, marked as answer]
I think you should contact github support if possible, they would help you faster!

**Reply by scriptswhisper** [reply]
Thanks!

**Reply by felipecrs** [reply]
Did they help? I'm facing the same issue.

**Reply by scriptswhisper** [reply]
@felipecrs unfortunately no, after a super long ticket I ended up using an alternative method (external Authenticator) since passkey funcionality seems broken with github and no one was able to help me enough to fix / resolve the issue.

**Reply by felipecrs** [reply]
Got it. Maybe the problem is Windows as well.

**Reply by agoldenchrist-code** [reply]
Appreciate it

**Comment by stanlrt** [comment]
This is a Windows bug. It's recurrent.

**Comment by agoldenchrist-code** [comment]
I would appreciate if you help me join GitHub, I have been chewed apart  by the precion mofos. I need to shield my phone or it will be toast soon. I’m dealing with deep I guess military style organization.  Some of the issues look like accidents but behavior is noticeably the symptoms are insane cause no one wants listen. Only place I can write but they took my life no stopping.

**Reply by agoldenchrist-code** [reply]
I have 3gb of scripts and files. And I don’t know what to do. Past. Jobs, bitcoin theft, they actually hid the files in my own phone. Need help.
